### PR TITLE
Regression: API CORS not working after Cordova being disabled by default

### PR DESCRIPTION
**Pull request by rodrigok**

### CODE REVIEW

File: app/api/server/api.js [hunk 1]
```diff
@@ -599,15 +610,26 @@ const createApi = function _createApi(enableCors) {
...
 	}
 };
 
 // also create the API immediately
 createApi(!!settings.get('API_Enable_CORS'));
 
 // register the API to be re-created once the CORS-setting changes.
-settings.get('API_Enable_CORS', (key, value) => {
-	createApi(value);
+settings.get(/^API_Enable_CORS|API_CORS_Origin$/, () => {
```
Comment (tassoevan) [inline app/api/server/api.js]:
```suggestion
settings.get(/^(API_Enable_CORS|API_CORS_Origin)$/, () => {
```

File: app/api/server/api.js [hunk 2]
```diff
@@ -588,9 +588,20 @@ const createApi = function _createApi(enableCors) {
 			defaultOptionsEndpoint,
 			auth: getUserAuth(),
 		});
+	} else {
+		API.v1._config.enableCors = enableCors;
+		if (enableCors) {
```
Comment (sampaiodiego) [inline app/api/server/api.js]: to reduce duplicated code I'd recommend creating a function to do what [this block of code](https://github.com/RocketChat/Rocket.Chat/pull/15443/files#diff-e8a96717dbd3869af6e51ee232c94025R593-R601) does.

File: app/api/server/api.js [hunk 3]
```diff
@@ -599,15 +610,26 @@ const createApi = function _createApi(enableCors) {
...
 	}
 };
 
 // also create the API immediately
 createApi(!!settings.get('API_Enable_CORS'));
 
 // register the API to be re-created once the CORS-setting changes.
-settings.get('API_Enable_CORS', (key, value) => {
-	createApi(value);
+settings.get(/^(API_Enable_CORS|API_CORS_Origin)$/, () => {
```
Comment (sampaiodiego) [inline app/api/server/api.js]:
```suggestion
settings.get(/^(API_Enable_CORS|API_CORS_Origin)$/, (key, value) => {
```

File: app/api/server/api.js [hunk 4]
```diff
@@ -599,15 +610,26 @@ const createApi = function _createApi(enableCors) {
...
+	createApi(settings.get('API_Enable_CORS'));
```
Comment (sampaiodiego) [inline app/api/server/api.js]:
```suggestion
	createApi(value);
```

**Pull request merged**